source/module_cell/klist.h

```diff
@@ -13,6 +13,7 @@ class K_Vectors
 public:
     std::vector<ModuleBase::Vector3<double>> kvec_c; /// Cartesian coordinates of k points
     std::vector<ModuleBase::Vector3<double>> kvec_d; /// Direct coordinates of k points
+    std::vector<ModuleBase::Vector3<double>> kvec_c_full; // Cartesian coordinates of full k mesh match with nkstot_full
```

Comment:
what' the definition of 'full' in this case?

Reply:
The current implementation of the Ewald summation for calculating the EXX Vq term does not support symmetry operations, requiring the use of the full k-point grid without symmetry reduction. This "full" keyword can be removed once symmetry support is implemented in future updates.

Reply:
I would suggest you open a local array to store all kvec_c_full data in the code where you need it, do not add new iterms in klist.h, because your code will introduce uncessary k-point arryas for other calculations.